BossMod/AI/AIManager.cs

```diff
@@ -7,11 +7,11 @@
 
 namespace BossMod.AI
 {
-    class AIManager : IDisposable
+    public class AIManager : IDisposable
     {
-        private Autorotation _autorot;
+        public Autorotation _autorot;
```

Comment:
The plugin follows standard c# naming conventions (private fields _likeThis, public LikeThis). See https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names